packages/@uppy/form/src/index.js

```diff
@@ -12,6 +12,9 @@ import packageJson from '../package.json'
 export default class Form extends BasePlugin {
   static VERSION = packageJson.version
 
+  /** @type {HTMLFormElement} */
+  form
```

Comment:
This should probably not be public
```suggestion
  form // TODO: make this private (or at least, mark it as readonly)
```